src/toPropertyPath.ts

```diff
@@ -0,0 +1,9 @@
+import { decodePointerFragment } from './decodePointerFragment';
+
+export function toPropertyPath(path: string) {
+  return path
+    .replace(/^\//, '')
```

Comment:
This is good. But if you take a look at https://github.com/stoplightio/json/blob/master/src/pathToPointer.ts#L16 you'll see that some of our functions that generate JSON pointers add a leading `#` (yay URL syntax). So let's go ahead and make this `.replace(/^(\/|#\/)/, '')` so it can handle these types of pointers as well.